src/libsync/creds/oauth.h

```diff
@@ -59,7 +59,7 @@ class OWNCLOUDSYNC_EXPORT OAuth : public QObject
     OAuth(Account *account, QObject *parent);
     ~OAuth();
     void startAuthentification();
-    void refreshAuthentification(const QString &refreshToken);
+    void refreshAuthention(const QString &refreshToken);
```

Comment:
I did not know Authention is a word.  What's the reason behind this  change?

Reply:
Maybe it should be "Authentication" ?

Reply:
thx 🤦‍♀